conda_requirements.txt

```diff
@@ -0,0 +1,27 @@
+# requirements file for conda
```

Comment:
Do we need this if the dependencies are declared in `pyproject.toml correctly?

Reply:
If everyone is using pixi, then this is totally unneeded.

I put this in before I did the pixi stuff.

So probably delete it, and if we need a way to specify conda requirements outside of pixi, we can auto-generate from the pyproject.toml.